pandas/io/auth.py

```diff
@@ -30,7 +30,7 @@ class AuthenticationConfigError(ValueError):
 
    %s
 
-with information from the APIs Console <https://code.google.com/apis/console>.
+with information from the APIs Console <https://console.developers.google.com/iam-admin/projects>.
```

Comment:
This gives a lint error (line to long), that's why travis is failing. Can you put the link on a new line?